src/main/java/org/tailormap/api/persistence/helper/ApplicationHelper.java

```diff
@@ -418,5 +427,30 @@ private Triple<GeoService, GeoServiceLayer, GeoServiceLayerSettings> findService
       GeoServiceLayerSettings layerSettings = service.getLayerSettings(layerRef.getLayerName());
       return Triple.of(service, serviceLayer, layerSettings);
     }
+
+    private boolean isWebMercatorAvailable(AppTreeLayerNode appTreeLayerNode) {
+      Triple<GeoService, GeoServiceLayer, GeoServiceLayerSettings> serviceWithLayer =
+          findServiceLayer(appTreeLayerNode);
+      GeoServiceLayer serviceLayer = serviceWithLayer.getMiddle();
+      GeoService service = serviceWithLayer.getLeft();
+
+      if (service != null && service.getProtocol() == XYZ) {
```

Comment:
null checks beter lostrekken ipv elke keer herhalen, zoals in bv addAppLayerItem():

```java
      if (service == null || serviceLayer == null) {
        return false;
      }
```